.github/schema-loading.yaml

```diff
@@ -39,7 +39,7 @@ spec:
     spec:
       containers:
       - name: schema-loading
-        image: ghcr.io/scalar-labs/scalardb-schema-loader:3.5.2
+        image: ghcr.io/scalar-labs/scalardb-schema-loader:4.0.0-SNAPSHOT
```

Comment:
We are planning to use SNAPSHOT version containers in the `main` branch and CI in the future. So, I also use the SNAPSHOT version here.